File: APTJava/app/src/main/java/com/binyouwei/apt_java/MainActivity.java
package com.binyouwei.apt_java;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

import com.example.apt_compiler.annotation.BindView;
import com.example.apt_compiler.launcher.AutoBind;

public class MainActivity extends AppCompatActivity {

    @BindView(value = R.id.test_textview)
    public TextView testTextView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        AutoBind.getInstance().inject(this);

        testTextView.setText("APT 测试");

    }
}

File: APTKotlin/app/src/main/java/com/binyouwei/aptkotlin/MainActivity.kt
package com.binyouwei.aptkotlin

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.example.apt_compiler.annotation.BindView
import com.example.apt_compiler.launcher.AutoBind

class MainActivity : AppCompatActivity() {

    @BindView(value = R.id.test_textview)
    var text: TextView? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        AutoBind.inject(this)

        text?.text = "APT 测试"
    }
}